src/annotator/guest.js

```diff
@@ -593,8 +589,8 @@ export default class Guest {
    */
   _updateAnchors(anchors, notify) {
     this.anchors = anchors;
-    if (notify) {
-      this._emitter.publish('anchorsChanged', this.anchors);
+    if (notify && this._frameIdentifier === null) {
```

Comment:
Special treatment of the `Guest` that is in the `host` frame. However, this assumption could be lifted in the future.